arangod/Pregel/Status/Status.h

```diff
@@ -115,7 +115,7 @@ template<typename Inspector>
 auto inspect(Inspector& f, GraphStoreStatus& x) {
   return f.object(x).fields(f.field("verticesLoaded", x.verticesLoaded),
                             f.field("edgesLoaded", x.edgesLoaded),
-                            f.field("memoryUsed", x.memoryBytesUsed),
+                            f.field("memoryBytesUsed", x.memoryBytesUsed),
```

Comment:
Are there any users of this attribute that would need to be adjusted as well after the name change?

Reply:
No, because the field was just introduced 3 weeks ago: https://github.com/arangodb/arangodb/pull/16515